JSDemos/Demos/TabPanel/Overview/Angular/app/app.component.css

```diff
@@ -1,45 +1,122 @@
-::ng-deep .tabpanel-item {
-  height: 200px;
-  -webkit-touch-callout: none;
-  -webkit-user-select: none;
-  -khtml-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
-  user-select: none;
-  padding-left: 25px;
-  padding-top: 55px;
+::ng-deep .tabpanel-demo {
+  display: flex;
+  height: 100%;
+  min-height: 450px;
 }
 
-::ng-deep .mobile .tabpanel-item {
-  padding-top: 10px;
+::ng-deep .widget-container {
+  display: inline;
+  flex-grow: 1;
+  min-width: 450px;
+  padding: 16px 32px;
 }
 
-::ng-deep .tabpanel-item > div {
-  float: left;
-  padding: 0 85px 10px 10px;
+::ng-deep .dx-theme-material .dx-tabpanel {
+  border-radius: 8px;
+  overflow: hidden;
```

Comment:
Please explain why we should use `overlow: hidden` for `.dx-tabpanel` element. I don't understand it from the context.

Reply:
For using border-radius.